### ISSUE TITLE
Overlay composables on top of WebView

### ISSUE THREAD

**Issue by CodyTroopDev** [KevinnZou/compose-webview-multiplatform#162]
I want to have a fullscreen WebView in my application, but still allow controls to navigate to other screens etc.

When I attempt to place Buttons on top of the WebVew, they do not appear.  I was wondering if there was a setting I could use to change the behavior or possibly something else I had missed.  

If I need to, I can always add the buttons to the webpage itself, but I'd like to write fewer functions that cross the bridge if possible.

Here is my current code:
<img width="1082" alt="Screenshot 2024-05-24 at 10 52 09 AM" src="https://github.com/KevinnZou/compose-webview-multiplatform/assets/49536787/406b9d40-a01e-42e6-baed-25a25170bd91">

**Comment by KevinnZou** [comment]
@CodyTroopDev Yes, it is a known issue with Compose Multiplatform desktop. We can only wait for an official solution. I am sorry for that.

**Comment by anye6488** [comment]
> @CodyTroopDev Yes, it is a known issue with Compose Multiplatform desktop. We can only wait for an official solution. I am sorry for that.是的，这是 Compose 多平台桌面的一个已知问题。我们只能等待官方的解决方案。我对此感到抱歉。

Consider using off-screen rendering to use the Compose component for rendering, in some lightweight use cases?

<img width="800" alt="image" src="https://github.com/user-attachments/assets/dcc890e1-7e35-4cd1-9c1b-2b99c86f3b88" />
